homeassistant/components/notify/notifyme.py

```diff
@@ -0,0 +1,52 @@
+"""
+NotifyMe platform for notify component.
+
+NotifyMe is an Alexa Skill which allows you to send any message to your
+Amazon Echo devices via a simple REST API.
+
+You have to install and set up the skill first to acquire an access code
+before you can set up this platform. See http://www.thomptronics.com/notify-me
+for complete instructions.
+"""
+import logging
+import aiohttp
+
+import voluptuous as vol
+
+from homeassistant.const import (CONF_NAME, CONF_ACCESS_TOKEN)
+from homeassistant.components.notify import (
+    BaseNotificationService, PLATFORM_SCHEMA)
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_ACCESS_TOKEN): cv.string,
+    vol.Optional(CONF_NAME): cv.string,
+})
+
+NOTIFYME_API_ENDPOINT = "https://api.notifymyecho.com/v1/NotifyMe"
+
+
+def get_service(hass, config, discovery_info=None):
+    """Get the NotifyMe notification service."""
+    access_token = config[CONF_ACCESS_TOKEN]
```

Comment:
Would be nice if you check that the token is valid otherwise the users end-up with a non-working platform in their setup.

Reply:
Hi @fabaff , yes that would be nice indeed. Although I don't really see an API that I can call to verify the token without sending out an actual message to Alexa, this might be annoying?
http://www.thomptronics.com/notify-me

Reply:
We have had that issue for other platforms as well. For the `pushetta` platform we created a configuration key (`send_test_msg`) but I doubt that it is used.

Sending test messages in the middle of the night is not the way to go.

Reply:
Given there is no better way to check the token validity other than the "required" check by config schema, I'll keep it as is unless you feel otherwise :)